br/backup-and-restore-tool.md

```diff
@@ -29,7 +29,11 @@ Refer to the [download page](/download-ecosystem-tools.md#br-backup-and-restore)
 
 BR sends the backup or restoration commands to each TiKV node. After receiving these commands, TiKV performs the corresponding backup or restoration operations. Each TiKV node has a path in which the backup files generated in the backup operation are stored and from which the stored backup files are read during the restoration.
 
-### Backup principle
+![br-arch](/media/br-arch.png)
+
+<details>
```

Comment:
Where does this come from? It's not from https://github.com/pingcap/docs-cn/pull/3706/files? @TomShawn 

And are you sure that the frontend already supports this display in our new docs website?

Reply:
As confirmed with @YiniXu9506, this display is supported in our new docs website. See [Telemetry](https://docs.pingcap.com/tidb/dev/telemetry#disable-tidb-telemetry-at-deployment) for example. 

In https://github.com/pingcap/docs-cn/pull/3706/files, @overvenus removes the implementation part and adds a link to a Chinese page which cannot be added here. I think it is OK to retain the implementation part (given that there is no alternative to it elsewhere) and with the folding display, users can choose to see or not to see the lengthy implementation part, so I make the change in this PR as you see.